src/main/java/edu/harvard/iq/dataverse/DatasetFieldValueValidator.java

```diff
@@ -19,6 +19,7 @@
 import javax.validation.ConstraintValidator;
 import javax.validation.ConstraintValidatorContext;
 import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.validator.routines.UrlValidator;
```

Comment:
There's a similarly named edu.harvard.iq.dataverse.URLValidator class that looks like it was used in bean validation (pre v4.13) and in a couple places to validate URLs from settings. I don't see any place where it it used for displayed URLs. Never-the-less, seems like using the same Apache UrlValidator class there makes sense.

Reply:
Ok, have updated URLValidator to use org.apache.commons.validator.routines.UrlValidator, see 6de1e959555c315f79826e8ab86796a696aa49ee. It passed the corresponding unit test.